platform/viewer/.webpack/writePluginImportsFile.js

```diff
@@ -66,7 +67,23 @@ function getRuntimeLoadModesExtensions() {
     ' for(const modeFactory of modesFactory) {\n' +
     '  const newModes = await modeFactory(modes,extensions);\n' +
     '  newModes.forEach(newMode => modes.push(newMode));\n' +
-    '}\n}\n'
+    '}\n}\n\n\n' +
+    'async function loadRuntimeImports(config) {\n' +
+    '  if (config && config.modes && config.modes.length) {\n' +
+    '    for (const modeName of config.modes) {\n' +
+    '      const existingMode = modes.find(mode => mode.id === modeName);\n' +
+    '      if (!existingMode) {\n' +
+    '        if (modeName === `@ohif/mode-basic-test-mode`) {\n' +
```

Comment:
This should be generic code generated from the plugingConfig.json file dynamic entries list

Reply:
Will address in a separate PR as discussed